CHANGELOG.md

```diff
@@ -0,0 +1,298 @@
+### Changelog
+
+All notable changes to this project will be documented in this file. Dates are displayed in UTC.
+
+Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
+
+#### [4.0.0-pre.0](https://github.com/GitbookIO/nuts/compare/3.1.1...4.0.0-pre.0)
+
+- chore: Remove unused backends/github.js imports [`#18`](https://github.com/GitbookIO/nuts/pull/18)
```

Comment:
Links aren't quite right as [auto-changelog](https://www.npmjs.com/package/auto-changelog) is not accounting for the original and this fork (i.e. https://github.com/pzrq/nuts) on GitHub out of the box. Probably would be nice to teach it how do that in future though not a current priority.